docs/best-practices/index.rst

```diff
@@ -92,3 +92,10 @@ Starting from Autofac 5.x the container is immutable. There are several potentia
 In order to prevent any of these risks to become a problem, it is no longer an option to update the container after construction.
```

Comment:
I know this isn't a new change, but I can't unsee it.
```suggestion
In order to prevent any of these risks from becoming a problem, it is no longer an option to update the container after construction.
```